internal/context.go

```diff
@@ -284,3 +322,90 @@ func createContextFileIfNotExist() {
 		log.Verboseln("Created ~/.corectl/contexts.yml for storage of corectl contexts")
 	}
 }
+
+func getSessionCookie(tlsClientConfig *tls.Config, engineURL string, userName string, password string) string {
+	// Verify Qlik Sense URL
+	u, err := url.Parse(engineURL)
+
+	if err != nil {
+		log.Fatalln("The engineURL doesn't seem to be correct")
+	}
+
+	if u.Scheme != "https" {
+		log.Fatalln("We only support login through secure connections (HTTPS)")
```

Comment:
Think "We" should be skipped. Perhaps `Only login through secure connections (HTTPS) is supported`?

Reply:
Yes sir